fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/domain/SearchQueryRenderer.java

```diff
@@ -270,8 +281,8 @@ public QueryData getParameterBaseQuery(QueryData parent) {
         //           AND LRn.LOGICAL_RESOURCE_ID = LRp.LOGICAL_RESOURCE_ID
         SelectAdapter exists = Select.select("1");
         exists.from(xxLogicalResources, alias(lrAlias))
-         .where(lrAlias, "IS_DELETED").eq().literal("N") // TODO remove either from here or parent
-         .and(lrAlias, "LOGICAL_RESOURCE_ID").eq(parentLRAlias, "LOGICAL_RESOURCE_ID"); // correlate to parent query
+           .where(lrAlias, "IS_DELETED").eq().literal("N") // TODO remove either from here or parent
```

Comment:
looks like we only touched this line to fix the formatting, but now I'm wondering about this TODO...

Reply:
This method gets called by multiple parents, so I'm not comfortable pulling it out. Certainly worth looking at though.

Reply:
I re-looked at this and verified that we could remove it, and went ahead and removed it and left a comment that it's the caller's responsibility to check IS_DELETED if necessary.